core/src/main/java/org/eclipse/hono/util/CredentialsConstants.java

```diff
@@ -34,6 +34,10 @@
      * The name of the field that contains the secret(s) of the credentials.
      */
     public static final String FIELD_SECRETS                     = "secrets";
+    /**
+     * The name of the field that contains the extension fields.
+     */
+    public static final String FIELD_EXT                         = "ext";
```

Comment:
the `CredentialsConstants` class is used to define the constants required for interacting with and implementing the Credentials API.
In the past, when we deliberately used the Credential API's content (payload) model for the management API as well, all the constants defined here were also used for implementing the management API.
However, with the clear distinction between the Credentials API and the HTTP management API, I would rather define the constants required for the HTTP management API in separate classes and not *mix* them with each other. That way we would remove all (unwanted) dependencies between these two APIs and would be free to evolve them separately without the need for refactoring the constant names and values.

WDYT?

Reply:
Are you thinking also of duplicating all constants that are shared between two APIs?

Reply:
If we want a clean separation between the two, then I think we will have to, don't you think?
The fact that many of the fields used in the payload of the two APIs have the same name needs to be considered arbitrary/coincidental, doesn't it?

Reply:
OK, sounds good to me. I'll do some refactoring in this area. Thanks